packages/client/hmi-client/src/types/Project.ts

```diff
@@ -1,37 +1,8 @@
-import {
-	Artifact,
-	AssetType,
-	Code,
-	Dataset,
-	DocumentAsset,
-	ExternalPublication,
-	Model,
-	Project
-} from '@/types/Types';
-import { Workflow } from '@/types/workflow';
+import { AssetType } from '@/types/Types';
```

Comment:
This entire class i think can go away once this is done. That `isProjectAssetTypes` is only used in one place and i think its only for the deprecated assettype.. And that enum of one value seems unused?

Reply:
That is the goal